<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inventory</class>
 <widget class="QMainWindow" name="inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SalePulse:Sales Inventory system</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main Background with Dotted Pattern */
QWidget {
    background-color: #0a192f; /* Navy Blue */
    background-image: radial-gradient(circle at 50% 50%, 
        rgba(255,255,255,0.1) 1.5px, 
        transparent 2px);
    background-size: 20px 20px;
    color: #ffffff;
    font-family: 'Roboto', sans-serif;
}

/* Glowing Accents */
QPushButton {
    background-color: rgba(16, 42, 74, 0.8); /* Dark Blue Glass */
    border: 1px solid #00b4d8; /* Cyan Border */
    border-radius: 6px;
    padding: 10px 25px;
    color: #ffffff;
    font-weight: 500;
    transition: all 0.3s ease;
    backdrop-filter: blur(5px);
}

QPushButton:hover {
    background-color: #00b4d8; /* Cyan */
    box-shadow: 0 0 15px rgba(0, 180, 216, 0.5);
    transform: scale(1.05);
}

QPushButton:pressed {
    background-color: #0096c7; /* Darker Cyan */
}

/* Futuristic Input Fields */
QLineEdit, QTextEdit, QSpinBox {
    background-color: rgba(16, 42, 74, 0.6);
    border: 2px solid #48cae4; /* Light Cyan */
    border-radius: 4px;
    padding: 8px;
    color: #ffffff;
    selection-background-color: #00b4d8;
}

QLineEdit:focus, QTextEdit:focus, QSpinBox:focus {
    border-color: #00b4d8;
    box-shadow: 0 0 12px rgba(0, 180, 216, 0.3);
}

/* Star-like Dots Enhancement */
QWidget#mainContent {
    background-image: radial-gradient(circle at 50% 50%, 
        rgba(255,255,255,0.3) 1px, 
        transparent 2px),
        radial-gradient(circle at 20% 80%, 
        rgba(255,255,255,0.2) 1px, 
        transparent 3px);
    background-size: 50px 50px, 80px 80px;
}

/* Holographic Cards */
QFrame.card {
    background: rgba(16, 42, 74, 0.7);
    border: 1px solid rgba(72, 202, 228, 0.3);
    border-radius: 8px;
    padding: 15px;
    backdrop-filter: blur(10px);
}

/* Sci-Fi Scrollbar */
QScrollBar:vertical {
    background: rgba(16, 42, 74, 0.5);
    width: 12px;
    border-radius: 6px;
}

QScrollBar::handle:vertical {
    background: #00b4d8;
    border-radius: 6px;
    min-height: 30px;
}

/* Cyber SpinBox */
QSpinBox {
    padding-right: 30px;
}

QSpinBox::up-button, QSpinBox::down-button {
    background-color: #023e8a;
    border: 1px solid #48cae4;
    width: 25px;
    border-radius: 4px;
}

QSpinBox::up-arrow {
    image: url(path/to/cyber-up-arrow.svg);
}

QSpinBox::down-arrow {
    image: url(path/to/cyber-down-arrow.svg);
}

/* Animated Progress Bar */
QProgressBar {
    border: 2px solid #023e8a;
    border-radius: 5px;
    text-align: center;
}

QProgressBar::chunk {
    background-color: qlineargradient(
        x1:0, y1:0,
        x2:1, y2:0,
        stop:0 #00b4d8, 
        stop:1 #03045e
    );
    border-radius: 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="pro_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Enter Product Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="add_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Add Product</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="sell_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>200</horstretch>
            <verstretch>30</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="cost_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="sell_pri">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Sell Price</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="cost_pri">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Cost Price</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="pro_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="next_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>150</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>12</pointsize>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="quantity_n">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Quantity</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="checkout_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>12</pointsize>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Checkouts</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="cus_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Customer Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="addc_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Add Cart</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="nex_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>12</pointsize>
            <fontweight>Medium</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="product_n">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Product Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="cus_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="product_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="value_ed">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLabel" name="revenue_n">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Total Revenue</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="revenue_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="profit_n">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="exits_btn">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Exits</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="t_profit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>13</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Total Profit</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="calculate_btn">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAdd_prooduct">
   <property name="text">
    <string>Add prooduct</string>
   </property>
  </action>
  <action name="actionCustmer_checkouts">
   <property name="text">
    <string>Customer checkouts</string>
   </property>
  </action>
  <action name="actionCalculation_Revenue">
   <property name="text">
    <string>Calculation Revenue</string>
   </property>
  </action>
  <action name="actionExits">
   <property name="text">
    <string>Exits</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
